.github/workflows/publish.yml

```diff
@@ -8,10 +8,15 @@ on:
   push:
   pull_request:
 
+permissions:
+  contents: read
+
 jobs:
 
   test_wheel_building:
     # This ensures that a couple of targets work fine in pull requests and pushes
+    permissions:
```

Comment:
@WilliamJamieson , I didn't notice you do this part. Did you?

Reply:
Darn, no